warp10/src/main/java/io/warp10/standalone/Warp.java

```diff
@@ -214,6 +218,22 @@ public static void main(String[] args) throws Exception {
       }
     }
 
+    if (null != System.getProperty(Configuration.LEVELDB_BLOCK_SIZE)) {
+      options.blockSize(Integer.parseInt(System.getProperty(Configuration.LEVELDB_BLOCK_SIZE)));
+    }
+
+    if (null != System.getProperty(Configuration.LEVELDB_BLOCK_RESTART_INTERVAL)) {
+      options.blockRestartInterval(Integer.parseInt(System.getProperty(Configuration.LEVELDB_BLOCK_RESTART_INTERVAL)));
+    }
+
+    if (null != System.getProperty(Configuration.LEVELDB_VERIFY_CHECKSUMS)) {
+      options.verifyChecksums(Boolean.parseBoolean(System.getProperty(Configuration.LEVELDB_VERIFY_CHECKSUMS)));
```

Comment:
`"true".equals` is used for all other properties instead of `Boolean.parseBoolean`.